### ISSUE TITLE
install.sh does not install openag_python

### ISSUE THREAD

**Issue by gordonbrander** [OpenAgInitiative/openag_brain_install_rpi#4]
This means when you `rosrun openag_brain main` you get.

```
Traceback (most recent call last):
  File "/home/pi/catkin_ws/src/openag_brain/scripts/main", line 10, in <module>
    from openag.couch import Server
ImportError: No module named openag.couch
```

@LeonChambers is there a special reason we don't install openag_python and its dependencies?

**Comment by LeonChambers** [comment]
Nope, the script should be updated to install it

**Comment by yaromins** [comment]
Here we go https://github.com/OpenAgInitiative/openag_brain_install_rpi/pull/5

**Comment by LeonChambers** [comment]
Fixed in #5
